hugo.toml

```diff
@@ -202,8 +202,8 @@ no = 'Oh snap! Sorry to hear that. Please <a href="https://github.com/kairos-io/
   url = "/docs"
 
 [[params.versions]]
-  version = "v3.1.3"
-  url = "/v3.1.3/docs"
+  version = "v3.2.1"
```

Comment:
cant this be managed with renovate maybe?

Reply:
oh we already do but it needs your change to make it work better?

Reply:
we decided that this file has more changes that should all go in, in one PR (e.g. the k3s version), otherwise the links in docs will be broken. So we don't worry too much about automating it, because in the end, it will need manual intervention.